apps/src/templates/projects/ProjectCardGrid.jsx

```diff
@@ -134,21 +134,33 @@ const ProjectCardGrid = React.createClass({
         {(!this.state.showAll) &&
           <div>
             {this.state.showApp === 'playlab' &&
-              <ProjectAppTypeArea
-                labKey="playlab"
-                labName={i18n.projectTypeAllProjectsPlaylab()}
-                labViewMoreString={i18n.projectsViewAll()}
-                projectList={projectLists.playlab}
-                numProjectsToShow={numProjects}
-                galleryType={this.props.galleryType}
-                navigateFunction={this.viewAllProjects}
-                isDetailView={true}
-              />
+            <ProjectAppTypeArea
+              labKey="playlab"
+              labName={showMoreProjects ? i18n.projectGroupPlaylabAllProjects() : i18n.projectTypeAllProjectsPlaylab()}
+              labViewMoreString={i18n.projectsViewAll()}
+              projectList={projectLists.playlab}
+              numProjectsToShow={numProjects}
+              galleryType={this.props.galleryType}
+              navigateFunction={this.viewAllProjects}
+              isDetailView={true}
+            />
+            }
+            {this.state.showApp === 'events' &&
+            <ProjectAppTypeArea
+              labKey="events"
+              labName={i18n.projectGroupEventsAllProjects()}
+              labViewMoreString={i18n.projectsViewAll()}
+              projectList={projectLists.events}
+              numProjectsToShow={numProjects}
+              galleryType={this.props.galleryType}
+              navigateFunction={this.viewAllProjects}
+              isDetailView={true}
+            />
             }
             {this.state.showApp === 'artist' &&
               <ProjectAppTypeArea
                 labKey="artist"
-                labName={i18n.projectTypeAllProjectsArtist()}
+                labName={showMoreProjects ? i18n.projectGroupArtistAllProjects() : i18n.projectTypeAllProjectsArtist()}
```

Comment:
is it on purpose that some of these have a conditional and some of them don't?

Reply:
yep, only the ones that are changing as part of this overall public gallery change have conditionals, so for example App Lab and Game Lab have the same strings before and after so there is no conditional. Also, Minecraft didn't exist before as a category, so there is a conditional for whether it shows up but no conditional for its display name. The work item list describing this work is here, showing the "list of categories in order": https://docs.google.com/document/d/1NYgN4G3FDlyTevKGE4_5wNzYkB36BJKZzsLn_ESLGmA/edit#